pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -56,6 +61,7 @@ func NewCmdIPFailoverConfig(f *clientcmd.Factory, parentName, name string, out,
 			ErrOut: errout,
 		},
 		ImageTemplate:    variable.NewDefaultImageTemplate(),
+		ServiceAccount:   "ipfailover",
```

Comment:
we should deprecate --credentials at the same time, ensure the ipfailover pod uses service account credentials to contact the API, and stop setting the ca/cert/key envvars if no --credentials are passed